solr/modules/ltr/example/README.md

```diff
@@ -33,7 +33,15 @@ Please refer to the Solr Reference Guide's section on [Learning To Rank](https:/
 
   `ln -s /Users/YourNameHere/Downloads/liblinear-2.1 ./modules/ltr/example/liblinear`
 
-3. Extract features, train a reranking model, and deploy it to Solr.
+3. Prepare your Python3 environment to run the training script.
+
+   `./gradlew solr:modules:ltr:syncPythonClientSourceCode`
+
+   `pushd solrclient && python3 setup.py install --user && popd`
```

Comment:
pushd and popd are new to me!   at first I thought it was somethign I needed to understand...

Reply:
Alright, I've removed pushd/popd here as being perhaps too obscure.